### DISCUSSION TITLE
设置面板的一些建议

### DISCUSSION THREAD

**Discussion by JedSunCN** [the1812/Bilibili-Evolved#2672] [提问 / Q&A]
请问能不能将设置面板左边的选项菜单设置为常驻展开/折叠，否则鼠标放在上面就展开会遮挡右边的选项文字，且需要将鼠标移开一段距离后才能恢复，影响操作的便捷性与逻辑。

![QQ截图20211212133338](https://user-images.githubusercontent.com/41063651/145701671-9f652670-afe1-41ee-8c99-3f39701b17da.png)

**Comment by the1812** [comment]
移开后立即就会收起啊, 应该不影响看右边的文字的吧
